andela/ah-kgl-wakanda-frontend
# #166787732 Edit and delete an article

#### What does this PR do?
This PR will allow users to delete and edit their own articles. 

#### Description of Task to be completed?
- Adding an edit button on a single article that  will allow a user to go to the edit page
- Adding a delete button that will allow users to delete their articles

#### How should this be manually tested?
go to `one article view` and if you created the article you should see the edit and the delete button
 
#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?
[#166787732](https://www.pivotaltracker.com/story/show/166787732)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/27511264/60487933-df813c80-9ca0-11e9-90f7-5fca0fa4852c.png)

# 1 Delete and Edit Button
![image](https://user-images.githubusercontent.com/27511264/60511992-b332e380-9cd3-11e9-850c-5dd08735c411.png)


# 2 Edit 
![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/27511264/60511868-4cadc580-9cd3-11e9-95ff-ea0200a7c332.gif)

# 3 Delete
![ezgif com-video-to-gif (2)](https://user-images.githubusercontent.com/27511264/60511931-7666ec80-9cd3-11e9-88ee-f2ddec1e15d1.gif)




#### Questions:

N/A
